### ISSUE TITLE
Chrome doesn't load DataTables

### ISSUE THREAD

**Issue by FoXFTW** [SemanticMediaWiki/SemanticResultFormats#185]
### Setup

- MW version: 1.28.0
- DB (MySQL etc.): MariaDB 10.1.19
- PHP version: 7.0.13
- SMW version: 2.4.3
- SRF version: 2.4
- Browsers and versions (if applicable): newest Chrome 53 which is broken and Firefox 50 as working reference

### Issue

The DataTables format doesn't load in Chrome. The table will load on the first page load, after reloading the page the table won't load.
After clearing the local storage with:
```
window.localStorage.clear();
```
tables will load again, but only once.

StackTrace:
```
TypeError: Cannot read property 'aTypes' of undefined TypeError: Cannot read property 'aTypes' of undefined
    at eval (eval at <anonymous> (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:4), <anonymous>:171:1019)
    at eval (eval at <anonymous> (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:4), <anonymous>:172:303)
    at Object.<anonymous> (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:161)
    at fire (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:45)
    at Object.add [as done] (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:45)
    at Object.always (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:46)
    at runScript (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:161)
    at checkCssHandles (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:162)
    at execute (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:163)
    at Object.implement (load.php?debug=false&lang=de&modules=jquery%2Cmediawiki&only=scripts&skin=vector&version=0vdig2y:169)
```

Steps to reproduce the observation:
Go to https://sandbox.semantic-mediawiki.org/wiki/DataTables with Chrome and reload the page.

**Comment by kghbln** [comment]
Thanks for reporting. Indeed Chrome 53 gives pain here.

**Comment by mwjames** [comment]
> TypeError: Cannot read property 'aTypes' of undefined TypeError: Cannot read property

`debug=true` allows you to get more information about the actual cause.

Anyway, I checked in quick fix #186 which may address your issue but I cannot spent much time on this.

**Comment by kghbln** [comment]
> Anyway, I checked in quick fix #186 which may address your issue but I cannot spent much time on this.

I just pulled the new code to sandbox and can confirm that the datatables load even after purging the page.  Cool. Thanks a lot for the fix! @FoXFTW It will be great if you could do a local test at you end, too.

**Comment by FoXFTW** [comment]
@kghbln @mwjames #186 fixed the issue, thanks!

**Comment by kghbln** [comment]
@FoXFTW Thanks for testing. Will be released with 2.4.1 on upcoming Tuesday!
